dataduct/steps/count_check.py

```diff
@@ -30,7 +32,8 @@ def __init__(self, id, source_host, source_sql=None, source_table_name=None,
             **kwargs(optional): Keyword arguments directly passed to base class
         """
 
-        if not exactly_one(destination_table_name, destination_sql):
+        if not exactly_one(destination_table_name, destination_sql,
+                           destination_table_definition):
             raise ETLInputError('One of dest table or dest sql needed')
```

Comment:
I would improve this error message to mention that it could be a name or definition of a table.